editor/settings_config_dialog.h

```diff
@@ -75,6 +77,7 @@ class EditorSettingsDialog : public AcceptDialog {
 	Tree *shortcuts;
 	InputEventConfigurationDialog *shortcut_editor;
 	String shortcut_being_edited;
+	bool editing_primary_shortcut;
```

Comment:
```suggestion
	bool editing_primary_shortcut = false;
```
All primitive members should be initialized. (same with `button_id` above)